apis/Google.Storage.V1/Google.Storage.V1/StorageClientImpl.cs

```diff
@@ -34,7 +34,7 @@ namespace Google.Storage.V1
     public sealed partial class StorageClientImpl : StorageClient
     {
         private static readonly object _applicationNameLock = new object();
-        private static string _applicationName = "google-cloud-dotnet";
+        private static string _applicationName = UserAgentHelper.GetDefaultUserAgent(typeof(StorageClient));
```

Comment:
It's the application name as far as the REST library is concerned - it's used as the `BaseClientService.ApplicationName` property, but *we* know it's going to be used as the user agent.

I agree it's not totally ideal, but I'm not sure what would be better, to be honest.